### ISSUE TITLE
Argument of type 'unknown' is not assignable to parameter of type 'boolean | undefined'.

### ISSUE THREAD

**Issue by schoening** [oakserver/oak#340]
I wanted to jump right into deno with this tutorial: https://www.sanity.io/guides/server-side-rendering-deno-react
But I am getting the following error:

```
error: TS2345 [ERROR]: Argument of type 'unknown' is not assignable to parameter of type 'boolean | undefined'.
  Type 'unknown' is not assignable to type 'true'.
    const context = new Context(this, contextRequest, secure);
```
My app code:
```
// frontend/ssr/server.tsx

import { Application, Router } from 'https://deno.land/x/oak@v7.3.0/mod.ts';

const app = new Application();
const port: number = 8000;

const router = new Router();
router.get("/", (context) => {
    context.response.body =
        `<!DOCTYPE html>
				 <html lang="en">
				 <head>
			   <meta charset="UTF-8">
			   <meta name="viewport" content="width=device-width, initial-scale=1.0">
				 <link href="https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css" rel="stylesheet">
			   <title>Sanity <-> Deno</title>
				 </head>
				 <body>
			   <div id="root">
	       <h1> Deno and React with Sanity</h1>
				 </div>
				 </body>
				 </html>`;
});

app.use(router.routes());
app.use(router.allowedMethods());

app.listen({port});
console.log(`server is running on port: ${port}`);
```

**Comment by schoening** [comment]
I patched it for using the tutorial by removing the `boolean` union in `secureOrConn`. But I do not know if enough about the project to know if I should create a PR. Seems like the boolean was there for a reason?

**Comment by kitsonk** [comment]
What version of Deno are you using?

**Comment by schoening** [comment]
@kitsonk 

```
deno 1.11.0 (release, x86_64-unknown-linux-gnu)
v8 9.1.269.35
typescript 4.3.2
```

**Comment by kitsonk** [comment]
Ok, it looks like it was a "logic" error in previous versions of oak that made it incompatible with TypeScript 4.3 which shipped in Deno 1.11. (specifically https://github.com/oakserver/oak/commit/fe7ad3c5ea76b62be055c223c98ee748461f5937)

Updating to oak 7.5.0 appears to resolve the issue:

```
import { Application, Router } from "https://deno.land/x/oak@v7.5.0/mod.ts";
```

**Comment by schoening** [comment]
Great! Thank you for solving my issue 😃
